pontis-ui/src/components/Hero.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import Header from "./Header";
+
+const Hero = () => {
+  return (
+    <section className="w-screen bg-slate-50 py-40 px-6 sm:px-24 md:px-32">
+      <Header
+        title="The zk-Oracle"
+        subtitle="Pontis"
```

Comment:
My preference would be to have Pontis be the title and "the zk-oracle" be the subtitle